codecov.yml

```diff
@@ -0,0 +1,17 @@
+codecov:
+  branch: master
+coverage:
```

Comment:
Great feedback @okonet!!

I will check it out, will be handier when we have the checklist and the coverage automated and also running locally.